engine/storage/image/src/org/apache/cloudstack/storage/image/TemplateServiceImpl.java

```diff
@@ -687,7 +681,7 @@ public Void deleteTemplateCallback(AsyncCallbackDispatcher<TemplateServiceImpl,
         return future;
     }
 
-    protected Void syncTemplateCallBack(AsyncCallbackDispatcher<TemplateServiceImpl, CopyCommandResult> callback, TemplateOpContext<TemplateApiResult> context) {
+    protected void syncTemplateCallBack(AsyncCallbackDispatcher<TemplateServiceImpl, CopyCommandResult> callback, TemplateOpContext<TemplateApiResult> context) {
```

Comment:
Changing this creates compilation errors in caller.setCallback(...) in deleteTemplateAsync.
So the reason that Void is used, is that the response is passed to a method expecting an Object as parameter.